cmd/tarocli/assets.go

```diff
@@ -64,6 +65,11 @@ var mintAssetCommand = cli.Command{
 			Usage: "if true, then the asset supports on going " +
 				"emission",
 		},
+		cli.StringFlag{
+			Name: assetGroupKeyName,
+			Usage: "the specific group key to use to mint the " +
+				"asset. Must be used with --enable_emission",
```

Comment:
I think based on some of the past bug reports, we need to re-work the `--enable_emission` flag, but what's the rationale for also requiring that flag? IIUC, the original idea was that you'd do the enable emission the first time around, then the mints after that just need to reference that group key to signal it should be part of the asset group.

Reply:
roas is the suggestion:
>Since `--enable_emission` only influences the asset properties during the minting process, it's superfluous during subsequent steps.  Hence, remove any checks of it because the daemon should be interpreting whether or not the asset has permitted emissions

Reply:
Flipped it so it's required to be disabled, works out better than ignoring it entirely.